eng/common/templates/post-build/channels/public-validation-release.yml

```diff
@@ -48,7 +47,7 @@ stages:
           filePath: eng\common\sdk-task.ps1
           arguments: -task PublishArtifactsInManifest -restore -msbuildEngine dotnet 
             /p:ChannelId=$(PublicValidationRelease_30_Channel_Id)
-            /p:ArtifactsCategory=$(_DotNetArtifactsCategory)
+            /p:ArtifactsCategory=$(_DotNetValidationArtifactsCategory)
```

Comment:
@JohnTortugo @riarenas isn't this a breaking change? I'm setting _DotNetArtifactsCategory in my yml because I want to publish to my own specific feed. Will that still work after this change?

Reply:
This change only affects repos that publish to the Maestro Validation channel. I believe IOT doesn't publish there, right?

Reply:
This change just made it so there's two variables, one for the dev channel and one for the validation channel. They used to use the same variable. Your build sets the variable for the dev channel, which is the default channel that you publish to.

Reply:
oh I see, thanks for explaining. I do only use one channel so I guess this is fine then. Thanks!